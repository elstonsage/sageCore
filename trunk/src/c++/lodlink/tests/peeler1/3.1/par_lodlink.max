
 LINKAGE ANALYSIS BETWEEN LOCUS m1       AND LOCUS m2      
 ==========================================================


 --------------------------------------------------------------------------------------------------------------------------------
 --------------------------------------------------------------------------------------------------------------------------------
 MAXFUN
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.
 
 TOTAL # PARAMETERS IN THE MODEL NT =  2; # PARAMETERS TO BE ESTIMATED NE =  2; # DEPENDENT PARAMETERS ND =  1
 MAXIMUM # ITERATIONS MAXIT = 999
 CONTROL VALUES:
    IXVC = 0; IHIT = 0;
    EPSD =  .100E-01; YOTA =  .169E-05; EPST =  .130E-01; EPSC1 =  .100E-01; EPSC2 =  .100E-02; EPSC3 =  .100E-02
 --------------------------------------------------------------------------------------------------------------------------------
0METHOD 2:  DIRECT SEARCH METHOD, WITHOUT 2**(NI)-TRIAL SEARCH
0INITIAL CONDITIONS
    PARAMETER                           LOWER BOUND      INITIAL EST      UPPER BOUND     IN STEP FACT       STATUS
     1                                    .00000000        .50000001E-01    .50000000        .50000001E-01   IND-FN, MAY VARY
     2                                    .00000000        .50000001E-01    .50000000        .50000001E-01   DEPENDENT       
    INITIAL FUNCTION VALUE  -.1043256466E+02
 --------------------------------------------------------------------------------------------------------------------------------
0BASIC ITERATION PROCESS STOPPED AFTER ITERATION   3 DUE TO NEGLIGIBLE DIFFERENCE IN COMPUTED FUNCTION VALUES     .7815553465E-03
 (CONVERGENCE CRITERION 3)
0PARAMETER  1 FIXED AT BOUND    .00000000    
0AFTER ITERATION   2, ALL INDEPENDENT PARAMETERS CONVERGED TO BOUNDS
0FINAL CONDITIONS
    PARAMETER                             FINAL EST          STD DEV      FIRST DERIV       STATUS
     1                                    .00000000       ----------       ----------       IND-FN, FIXED BY MAXFUN AT BOUND    
     2                                    .00000000       ----------       ----------       DEPENDENT                           
    FINAL FUNCTION VALUE  -.1038127137E+02
    NUMBER OF FUNCTION EVALUATIONS    2

 --------------------------------------------------------------------------------------------------------------------------------
 --------------------------------------------------------------------------------------------------------------------------------

 --------------------------------------------------------------------------------------------------------------------------------
 --------------------------------------------------------------------------------------------------------------------------------
 MAXFUN
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.
 
 TOTAL # PARAMETERS IN THE MODEL NT =  2; # PARAMETERS TO BE ESTIMATED NE =  2; # DEPENDENT PARAMETERS ND =  1
 MAXIMUM # ITERATIONS MAXIT = 999
 CONTROL VALUES:
    IXVC = 2; IHIT = 0;
    EPSD =  .100E-01; YOTA =  .169E-05; EPST =  .130E-01; EPSC1 =  .100E-01; EPSC2 =  .100E-02; EPSC3 =  .100E-02
 --------------------------------------------------------------------------------------------------------------------------------
0METHOD 5:  VARIABLE METRIC METHOD, USING NEGATIVE IDENTITY FOR INITIAL HESSIAN
0INITIAL CONDITIONS
    PARAMETER                           LOWER BOUND      INITIAL EST      UPPER BOUND       STATUS
     1                                    .00000000        .00000000        .50000000       IND-FN, MAY VARY
     2                                    .00000000        .00000000        .50000000       DEPENDENT       
    INITIAL FUNCTION VALUE  -.1038127137E+02
 --------------------------------------------------------------------------------------------------------------------------------
0PARAMETER  1 FIXED AT BOUND    .00000000    
0AFTER ITERATION   2, ALL INDEPENDENT PARAMETERS CONVERGED TO BOUNDS
