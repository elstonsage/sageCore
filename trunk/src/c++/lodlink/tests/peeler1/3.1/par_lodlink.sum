1S.A.G.E. RELEASE 3.1 -- LODLINK   		DECEMBER 1997
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.


 LODLINK GENERAL PARAMETERS
 --------------------------
     TITLE ............................... Test 01                                                                         
     NUMBER OF MARKER LOCI ...............    2
     FEMALE AND MALE SYMBOLS ............. F  M
     MORTON'S HOMOGENEITY TEST ........... NO 
     SMITH'S HOMOGENEITY TEST AND TEST
     FOR LINKAGE UNDER HETEROGENEITY ..... NO 
     NUMBER OF PEDIGREES .................    1
     RECOMBINATION FRACTION 
      -- FOR MALES, (THETA M) ............   .0000    .0100    .0500    .1000    .2000    .3000    .4000
      -- FOR FEMALES, (THETA F) ..........   .0000    .0100    .0500    .1000    .2000    .3000    .4000
1S.A.G.E. RELEASE 3.1 -- LODLINK   		DECEMBER 1997
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.


 LINKAGE ANALYSIS BETWEEN LOCUS m1       AND LOCUS m2      
 ==========================================================

                            -------------------------------------------------------------------
 THETA M                    !     .0000    .0100    .0500    .1000    .2000    .3000    .4000 !
 THETA F                    !     .0000    .0100    .0500    .1000    .2000    .3000    .4000 !
 ----------------------------------------------------------------------------------------------
 !   PEDIGREE   NUMBER OF   !                                                                 !
 !   NUMBER     INDIVIDUALS ! LOD SCORES                                                      !
 ----------------------------------------------------------------------------------------------
 !   00001          5       !     .3010    .2967    .2788    .2553    .2041    .1461    .0792 !
 ----------------------------------------------------------------------------------------------
 !                  5       !     .3010    .2967    .2788    .2553    .2041    .1461    .0792 !
 ==============================================================================================

                                         LOD      CHI             P-VALUE
                             ESTIMATE   SCORE    SQUARE  UPPER BOUND   APPROX. *
                             ========  =======  =======  ===========  ==========
 EQUIVALENT NUMBER OF MEIOSES   1.000
 THETA IN (0, 1)                 .000     .301    1.386      .500000     .119516 (ONE-SIDED)
 VARIANCE OF THETA            -------
 COMPLEMENT (1 - THETA)         1.000  -INFIN.

 * NOTE:  APPROXIMATE P-VALUES ASSUME A LARGE EQUIVALENT NUMBER OF
          INFORMATIVE MEIOSES.
