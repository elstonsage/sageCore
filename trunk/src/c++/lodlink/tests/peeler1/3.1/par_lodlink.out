1S.A.G.E. RELEASE 3.1 -- LODLINK   		DECEMBER 1997
 COPYRIGHT (C) 1997 CASE WESTERN RESERVE UNIVERSITY.


 LODLINK GENERAL PARAMETERS
 --------------------------
     TITLE ............................... Test 01                                                                         
     NUMBER OF MARKER LOCI ...............    2
     FEMALE AND MALE SYMBOLS ............. F  M
     MORTON'S HOMOGENEITY TEST ........... NO 
     SMITH'S HOMOGENEITY TEST AND TEST
     FOR LINKAGE UNDER HETEROGENEITY ..... NO 
     NUMBER OF PEDIGREES .................    1
     RECOMBINATION FRACTION 
      -- FOR MALES, (THETA M) ............   .0000    .0100    .0500    .1000    .2000    .3000    .4000
      -- FOR FEMALES, (THETA F) ..........   .0000    .0100    .0500    .1000    .2000    .3000    .4000

 Locus m1      

 A   =   .5000, B   =   .5000, 
  No.   Genotype   Phenotype   Genotypic Frequency

    1   A   A        A/A              .250000
    2   A   B        A/B              .500000
    3   B   B        B/B              .250000

 Locus m2      

 C   =   .3000, D   =   .7000, 
  No.   Genotype   Phenotype   Genotypic Frequency

    1   C   C        C/C              .090000
    2   C   D        C/D              .420000
    3   D   D        D/D              .490000

 LINKAGE ANALYSIS BETWEEN LOCUS m1       AND LOCUS m2      
 ==========================================================


 FAMILY NUMBER    1 (ID. IS 00001)
 ---------------------------------

         Rm       Rf     Log10[L(R)]     LOD-SCORE
      .0000    .0000       -4.508529       .301030
      .0100    .0100       -4.512894       .296665
      .0500    .0500       -4.530805       .278754
      .1000    .1000       -4.554286       .255273
      .2000    .2000       -4.605439       .204120
      .3000    .3000       -4.663431       .146128
      .4000    .4000       -4.730378       .079181
      .5000    .5000       -4.809559       .000000
